docs/index.md

```diff
@@ -86,44 +119,43 @@ LitElement uses [lit-html](https://github.com/Polymer/lit-html) to define and re
 </div>
 </section>
 
+
 <section>
 <div class="wrapper">
-<div class="responsive-row center">
-<div style="max-width: 600px">
-
-LitElement is brought to you by developers on the Google Chrome team with the input of web developers at organizations big and small around the world. 
 
+<h1 class="title">Browser Compatibility</h1>
+<h2 class="description">PWA Starter Kit works in all major browsers (Chrome, Firefox, IE, Edge, Safari, and Opera). </h2>
```

Comment:
`<h2 class="description">PWA Starter Kit works in all major browsers`

Should be `LitElement` instead of `PWA Starter Kit`